migrations/20230113201828-create-tag-stats-materialized-view.ts

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+    CREATE MATERIALIZED VIEW IF NOT EXISTS "CollectiveTagStats" AS
+    SELECT DISTINCT unnest(tags) AS tag, COUNT(*) AS count FROM "Collectives" GROUP BY tag ORDER BY count DESC
```

Comment:
Let's maybe put the `FROM`, `GROUP BY` and `ORDER BY` to new lines to make that easier to read

Reply:
Fixed!